### PR TITLE
Cache: Fixed redis cache to use multi batch size config value

### PR DESCRIPTION
**Pull request by abbyssoul**
Signed-off-by: Ivan Ryabov <abbyssoul@gmail.com>

* [x] I added CHANGELOG entry for this change.
* [ ] Change is not relevant to the end user.

## Changes

Cache: fixed redis client not respecting `SetMultiBatchSize` config value.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ We use *breaking :warning:* to mark changes that are not backward compatible (re
 ## Unreleased
 
 ### Fixed
-
+- [#5893](https://github.com/thanos-io/thanos/pull/5893) Cache: fixed redis client not respecting `SetMultiBatchSize` config value.
```
Comment (hanjm) [inline CHANGELOG.md:14]: Please add the last line in this section.

**Review by hanjm — APPROVED** [review]
LGTM, thanks.

File: CHANGELOG.md (hunk 1 above)
Comment (abbyssoul) [reply CHANGELOG.md]: so just to confirm, the changelog line should be added last to the list of Fixes, is that correct?

Comment (yeya24) [reply CHANGELOG.md]: Yeah, fixed section is good

**Review by yeya24 — APPROVED** [review]
Thanks for the contribution!

**Pull request merged**